massdriver-application/outputs.tf

```diff
@@ -23,5 +23,5 @@ output "params" {
 
 output "id" {
   description = "Cloud ID for application IAM (AWS Role, GCP Service Account, Azure Service Account, etc)"
-  value       = mdxc_application_identity.main.id
+  value       = local.pplication_identity_id
```

Comment:
Typo and we should use a ternary here to block the dag otherwise the output will return immediately for anything that _does_ need need an app identity created